vocabs/personal-profile-for-accessibility.ttl

```diff
@@ -0,0 +1,79 @@
+@prefix rdf:          <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:         <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:      <http://purl.org/dc/terms/> .
+@prefix skos:         <http://www.w3.org/2004/02/skos/core#> .
+@prefix voaf:         <http://purl.org/vocommons/voaf#> .
+@prefix vann:         <http://purl.org/vocab/vann/> .
+@prefix dbr:          <https://dbpedia.org/resource/> .
+@prefix wikidata:     <https://www.wikidata.org/entity/> .
+@prefix ppfa:         <http://w3id.org/drk/vocabs/personal-profile-for-accessibility#> .
+
+ppfa:
+    a                             voaf:Vocabulary, skos:ConceptScheme ;
+    dcterms:title                 "Personal Profile for Accessibility Vocabulary"@en ;
+    dcterms:description           "A controlled vocabulary for identifying and describing personal profiles from a disability perspective, aimed at facilitating accessibility."@en ;
+    rdfs:label                    "Personal Profile for Accessibility Vocabulary"@en ;
+    rdfs:comment                  "A controlled vocabulary for identifying and describing personal profiles from a disability perspective, aimed at facilitating accessibility."@en ;
+    voaf:hasVersion               "0.0.1" ;
+    vann:preferredNamespacePrefix "ppfa" ;
+    vann:preferredNamespaceUri    "http://w3id.org/drk/vocabs/personal-profile-for-accessibility#" ;
+    dcterms:source                <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/#personalprofile> ;
+    rdfs:seeAlso                  <https://sozialhelden.github.io/a11yjson/describing-objects/0-model/> .
+
+# See also ppfa:visualImpairment
+ppfa:blindness
```

Comment:
Wouldn't it rather follow best practices to use uppercase?

Reply:
It should, indeed! Updated!